HW1/HW1Q2.py

```diff
@@ -0,0 +1,7 @@
+s = int(input('Введите количество сделанных журавликов: '))
+
+Katya = int(s/6*4)
+Petya = int(s/6)
+Seryozha = int(s/6)
```

Comment:
Можно обойтись операторами целочисленного деления
```suggestion
Katya = s // 6 * 4
Petya = Seryozha  = s // 6
```